index.html

```diff
@@ -49,6 +82,72 @@
 
       var selectedConfigs = new Set();
 
+      var versionDateMapping = new Map();
+
+      function navigateToVersionFromDate(date) {
+        date = new Date(date).toDateString();
+        version = versionDateMapping.get(date);
+        if(!version) {
+          return // if no version present, do not navigate
+        }
+        let params = new URLSearchParams(document.location.search);
+        params.delete("version");
+        params.append("version", version);
+        url = window.location.pathname + `?${params.toString()}`;
+        window.location.href = url;
+      }
+
+      function populateVersionOptions(data) {
+        data = JSON.parse(data)
+        let resp = Array.from(data);
+        resp.forEach((res)=>{
+          versionDateMapping.set(new Date(res.commit.committer.date).toDateString(), res.sha)
+        })
+        let datepicker = document.getElementById("versionDateTimePicker")
+        new tempusDominus.TempusDominus(datepicker, {
+          restrictions: {
+            enabledDates: Array.from(versionDateMapping.keys().map(ds => new Date(ds))),
+          },
+          display: {
+              viewMode: 'calendar',
+              components: {
+                clock: false,
+                hours: false,
+                minutes: false,
+                seconds: false,
+                useTwentyfourHour: undefined
+              },
+            },
+            localization: {
+              format: 'MM-dd-yyyy',
+            },
+          useCurrent: false,
+        })
+
+      }
+
+      function getSummaryURL(version='') {
+        if(!version) return "summary.json" // By default fetch latest configuration summary
+        return `https://raw.githubusercontent.com/oracle/kconfigs/${version}/docs/summary.json`
+      }
+
+      function updateVersion(invalid=false) {
+        let params = new URLSearchParams(document.location.search);
+        let sp = document.getElementById("heading")
+        if(invalid) {
+          sp.innerHTML = `Distribution Kernel Configs`
+          alert("Invalid Version. Please add a valid version")
+          params.delete("version")
+          url = window.location.pathname + (params.toString())?`?${params.toString()}`:''
+          window.location.href = url;
+          return
+        }
+        var version = params.get("version");
+        if(!version) return "gh-pages";
+        sp.innerHTML = `Distribution Kernel Configs (version: ${version})`
```

Comment:
A nit here is that the date picker shows only dates, but the heading still shows a SHA. At load time, you don't actually know what date the SHA corresponds to, but you can update it later. I tweaked this a bit, so that once you load the list of commits, we update the text.

Reply:
I see.
Yea, we should show date instead of version.